pkg/platform/model/cve.go

```diff
@@ -22,7 +22,7 @@ func FetchProjectVulnerabilities(auth *authentication.Auth, org, project string)
 	// This should be removed by https://www.pivotaltracker.com/story/show/176508740
 	if !auth.Authenticated() {
 		return nil, errs.AddTips(
-			locale.NewError("cve_needs_authentication", "You need to be authenticated in order to access vulnerability information about your project."),
+			locale.NewInputError("cve_needs_authentication"),
```

Comment:
It feels wrong to me to mark this as an input error in the model. This assumes that we invoked the model with user input, but that may not always be the case.

If this is within scope of the rollbar report I'd suggest communicating a type here and asserting it from the runner to then return an input error.

Reply:
You're right about that. Also, the cve runner actually errors out if you're not authenticated to begin with, so the changes around lack of authentication were superfluous. I've changed the model to return an `ErrProjectNotFound` error and check for it in the runner in order to raise an input error though. Thanks for the suggestion.